docs/redirects.md

```diff
@@ -0,0 +1,15 @@
+# Redirects
+
+If you need to set up a new redirect, you can do that in our notifications-broadcasts-infra repository.
+
+In there, head to `terraform/modules/govuk-alerts-website/files/redirects` and add your redirect to REDIRECTS constant, using the following format:
+
+```
+REDIRECTS = {"<uri of page you want to redirect from>": "<uri of page you want to redirect to>"}
+```
+
+Deploy-wise, you will need to do 3 deploys in quick succession:
```

Comment:
"quick succession" scares me.

How quick? What happens if it's slow, because you get distracted, have wifi failure, functional tests are flaky, etc? Do we need to worry about fastly caching timelines in this?

Reply:
well we have a double entry if we are stuck in the middle (old page and new page both exist) - alternative way would be to not have a redirect for a short while? Then we only need 2 PRs, not 3, and we never have a confusing-looking list, and it is unlikely someone will use saved up link in that time. What do you think?

Reply:
i think three deploys should stay the same. As a team we're used to doing multiple PRs for zero-downtimey-ness when dealing with API changes, celery tasks, etc. I don't think there are any downsides to things being in a mid-state for a little while while PRs are reviewed, tests run, etc.

Is there a risk to having duplicate pages with no redirects between them? That seems fine to me as long as the content still makes sense, but there's some things I don't know the answer to like: fastly caching, google search indexing robots scraping pages, etc. Thoughts?

I don't think anyone here expects things to take days and days, but when I see "quick" I guess I start worrying about co-ordinating concourse jobs, etc. Or should we panic if it was quick but then concourse crashed. etc. And I _think_ for this the answer is "no we shoudn't panic if it takes a bit longer or someone goes on lunch during the deploy etc"

Reply:
yeah, not quick as very time sensitive, quick as relatively quick - I added a bit more context to the guidance.